src/components/thumbnail-card/ThumbnailCardDetails.js

```diff
@@ -1,19 +1,42 @@
 // @flow
 import * as React from 'react';
+import Tooltip from '../tooltip';
+import { useIsContentOverflowed } from '../../utils/dom';
 
 type Props = {
     actionItem?: React.Element<any>,
     icon?: React.Node,
+    shouldShowTooltipOnTitleHover: boolean,
     subtitle?: React.Node,
     title: React.Node,
 };
 
-const ThumbnailCardDetails = ({ actionItem, icon, subtitle, title }: Props) => (
+const RenderTitle = (shouldShowTooltipOnTitleHover: boolean, title: React.Node) => {
```

Comment:
style nit: if you're not calling this is a component, don't capitalize the function name

Reply:
I had an error earlier where it was saying that renderTitle wasn't a functional component so it couldn't use hooks so I changed it to be capitalized.